### PR TITLE
resolve checkstyle errors #214

### PR DESCRIPTION
**Pull request by ridmal**
pull request for #214 issue.

### CODE REVIEW

**Comment by a5anka** [comment]
It seems all the files in the repo now have the DOS line ending. Can we please not do that and use Linux file endings. Maybe there is a setting in the IDE you need to set to keep the Linux line endings.

**Comment by ridmal** [comment]
ok . . :)  sorry for the inconvenience.

**Comment by ridmal** [comment]
I added changes to PR. Could you please review again :)

File: .project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (warunalakshitha) [inline .project]: Can we remove those eclipse generated files like .project,.classpath,  .settings etc. ?

Comment (ridmal) [reply .project]: ok i will add .project , .classpath and .setting in .gitignore file and commit again.

**Comment by ridmal** [comment]
I removed .classpath, .settings and .project file from the repo and add changes to the gitignore file.

File: modules/broker-amqp/src/test/java/org/wso2/broker/amqp/codec/flow/ChannelFlowManagerTest.java [hunk 2]
```diff
@@ -29,7 +29,9 @@
 import org.wso2.broker.amqp.codec.frames.ChannelFlow;
 
 import java.util.stream.IntStream;
-
+/**
+ * Test class for Channel Flow Manager .
```
Comment (a5anka) [inline modules/broker-amqp/src/test/java/org/wso2/broker/amqp/codec/flow/ChannelFlowManagerTest.java]: Shall we remove the additional space after the word Manager and add a new line between the doc comment start line and import statement? Shall we fix in the other places as well?

**Comment by a5anka** [comment]
Why did we modify the *.jks files?

**Comment by ridmal** [comment]
I think it modified it file ending type to LF. (Linux file type) . It changes because I set file type from the IDE settings to the whole project.

**Comment by a5anka** [comment]
Closing due to inactivity.

**Comment by ridmal** [comment]
I think it is under review process.

**Comment by a5anka** [comment]
I don't think we need to change *.jks files in this PR?

**Comment by indikasampath2000** [comment]
Hi Ridmal,

If you go to https://github.com/ballerina-platform/ballerina-message-broker/pull/216/files, it still showing that .jks files are changed and following files have conflicts. Could you please resolve conficts and update the PR? 

modules/broker-auth/src/main/java/org/wso2/broker/auth/AuthManager.java
modules/broker-core/src/main/java/org/wso2/broker/core/queue/Queue.java
modules/broker-core/src/main/java/org/wso2/broker/core/store/DbWriter.java
modules/broker-core/src/main/java/org/wso2/broker/core/store/SharedMessageStore.java
modules/broker-core/src/test/java/org/wso2/broker/core/ExchangeDaoImplTest.java

**Comment by Asitha** [comment]
Hi Ridmal,

Can you squash the commits,

You can use ```git rebase -i HEAD~14```
14 is the number of commits you want to edit. Follow the instructions on git prompt and combine the commits. Having a single commit for this PR is ideal in my opinion

**Comment by a5anka** [comment]
Hi @ridmal,

A similar fix has been merged to master in PR https://github.com/ballerina-platform/ballerina-message-broker/pull/334. Therefore closing this PR. Thank you for the contribution. We hope to receive more contributions from you in the future. Extremely sorry for delayed response.

**Pull request closed without merging**